frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -0,0 +1,119 @@
+import { GEO_PROPERTIES, IGeoSearchParams } from 'constants/API';
+import { ENVIRONMENT } from 'constants/environment';
+import { BBox, Feature } from 'geojson';
+import useDeepCompareEffect from 'hooks/useDeepCompareEffect';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import { LatLngBounds } from 'leaflet';
+import React from 'react';
+import { useLeaflet } from 'react-leaflet';
+import { PointFeature } from '../types';
+import PointClusterer from './PointClusterer';
+
+export type InventoryLayerProps = {
+  /** Latitude and Longitude boundary of the layer. */
+  bounds?: LatLngBounds;
+  /** Zoom level of the map. */
+  zoom: number;
+  /** Minimum zoom level allowed. */
+  minZoom?: number;
+  /** Maximum zoom level allowed. */
+  maxZoom?: number;
+  /** Search filter to apply to properties. */
+  filter?: IGeoSearchParams;
+  /** What to do when the point feature is clicked. */
+  onMarkerClick?: (point: PointFeature, position?: [number, number]) => void;
+};
+
+/**
+ * Get a new instance of a BBox from the specified 'bounds'.
+ * @param bounds The latitude longitude boundary.
+ */
+const getBbox = (bounds: LatLngBounds): BBox => {
+  return [
+    bounds.getSouthWest().lng,
+    bounds.getSouthWest().lat,
+    bounds.getNorthEast().lng,
+    bounds.getNorthEast().lat,
+  ] as BBox;
+};
+
+/**
+ * Displays the search results onto a layer with clustering.
+ * This component makes a request to the PIMS API properties search WFS endpoint.
+ */
+export const InventoryLayer: React.FC<InventoryLayerProps> = ({
```

Comment:
Instead of the `Map` and `MapView` components doing all the work for making requests for PIMS inventory, it has been all moved into this `InventoryLayer` component.  At some point if I have time I will look into creating a `InventoryTileLayer` so that the requests to the DB will be more performant and cacheable.